db/config/config.js

```diff
@@ -0,0 +1,25 @@
+require('dotenv').config()
+
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DBNAME,
+    host:process.env.DB_HOST,
+    dialect: 'postgres'
+  },
+  test: {
```

Comment:
Nice job Henry. I like the proactiveness you showed in creating a .env.sample file right away. However, the config.js file has the same configuration for all environments (development, test, and production). Can you configure it for the different environments, and equally make the necessary changes in the .env and .env.sample so that it would not have to be done later. Nice job by the way. Thanks

Reply:
Thank you for the feedback Emmanuel, I have just refactored the config.js file and set up all the necessary environments as requested, and also replicated changes in the .env.sample file. Kindly review again, thanks.